docs/setup/independent/install-kubeadm.md

```diff
@@ -220,9 +231,44 @@ systemctl enable kubelet && systemctl start kubelet
 
 {% endcapture %}
 
+{% capture coreos %}
+
+Install CNI plugins (required for most pod network):
+
+```bash
+CNI_VERSION="v0.6.0"
+mkdir -p /opt/cni/bin
+curl -L "https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-amd64-${CNI_VERSION}.tgz" | tar -C /opt/cni/bin -xz
+```
+
+Install `kubeadm`, `kubelet`, `kubectl` and add a `kubelet` systemd service:
+
+```bash
+RELEASE="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+
+mkdir -p /opt/bin
+cd /opt/bin
+curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
+chmod +x {kubeadm,kubelet,kubectl}
+
+cd "/etc/systemd/system/"
+curl -L "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/kubelet.service" | sed 's:/usr/bin:/opt/bin:g' > kubelet.service
```

Comment:
nit 1: I prefer always writing `curl -sSL`...
nit 2: can you just redirect it to the `/etc/systemd/system/kubelet.service` directly instead of `cd`-ing into the dir first to save a step?